app/services/municipality-list.ts

```diff
@@ -27,6 +27,24 @@ export default class MunicipalityListService extends Service {
     return this._municipalities;
   }
 
+  async governingBodies() {
+    const GoverningBodyClasssificationCodes = await this.store.query(
+      'governing-body-classification-code',
+      {
+        page: { size: 600 },
+        sort: 'label',
+      }
+    );
+
+    return GoverningBodyClasssificationCodes.filter(
+      (classificationCode, index, self) =>
+        self.findIndex((t) => t.label === classificationCode.label) === index
+    ).map((governingBody) => ({
+      id: governingBody.label,
+      label: governingBody.label,
+    }));
+  }
+
```

Comment:
I would either rename the service or create a new `governing-body-list` service. It seems odd to have a `governingBodies` in a municipality-list service.